README.md

```diff
@@ -4,29 +4,44 @@
 
 ## New versions of WorldWind released
  
-Web WorldWind 0.10.0 and WorldWind Java 2.2.0 are now available on GitHub. The new version of Web WorldWind addresses potential vulnerabilities in the code and underlying packages. The new version of WorldWind Java focuses on upgrading to Java 11 and JOGL 2.4 as well as a switch to Apache 2.0 license. WorldWind's API remains largely unchanged in this release and we are committed to maintaining a consistent API in future releases.
-More information on the release can be found at these links: [Web WorldWind 0.10.0](https://github.com/NASAWorldWind/WebWorldWind/releases) and [WorldWind Java 2.2.0](https://github.com/NASAWorldWind/WorldWindJava/releases).
+Web WorldWind 0.11.0 is now available on GitHub. The summary of changes is as follows:
+- Improvements to COLLADA 3D model support.
+  - Added ability to obtain locations of a click in a 3D model
+  - Added visualization support to COLLADA models with wrongly formatted normals.
+- Transparent placemark picking has been fixed.
+- Improvements to Well-Known Text format support.
+- Various fixes to example code files.
+ 
+Development environment changes:
+- Added Chrome and Firefox headless browsers and removed PhantomJS
+- Switched IDEs from WebStorm to Visual Studio Code
+- Updated development dependencies to latest versions
+
+WorldWind's API remains largely unchanged in this release and we are committed to maintaining a consistent API in future releases.
+
+More information on the release can be found at these links: [Web WorldWind 0.11.0](https://github.com/NASAWorldWind/WebWorldWind/releases).
 Please direct questions to our new email address: 
 
 arc-worldwind@mail.nasa.gov
 
 [![Build Status](https://travis-ci.com/NASAWorldWind/WebWorldWind.svg?branch=develop)](https://travis-ci.com/NASAWorldWind/WebWorldWind)
```

Comment:
This link doesn't seem to work.

Reply:
I changed the phrasing to convey that the link leads to the release list, which was the original intent.